### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by AntiPontifex**
See title

### CODE REVIEW

**Review by BellezaEmporium — APPROVED** [review]
LGTM.

**Comment by BellezaEmporium** [comment]
data/channels.csv
 146    "2STV.sn" must be derived from the channel name "2sTV" and the country code "SN". Fix is to call the channel "2STV".
 203    "4DmasNoticiasTV.py" must be derived from the channel name "4DMas Noticias TV" and the country code "PY". Fix is to call the channel "4Dmas Noticias TV".
 224    "4TVNews.in" must be derived from the channel name "4tv News" and the country code "IN". Fix is to call the channel "4TV News".

**Comment by AntiPontifex** [comment]
> data/channels.csv 146 "2STV.sn" must be derived from the channel name "2sTV" and the country code "SN". Fix is to call the channel "2STV". 203 "4DmasNoticiasTV.py" must be derived from the channel name "4DMas Noticias TV" and the country code "PY". Fix is to call the channel "4Dmas Noticias TV". 224 "4TVNews.in" must be derived from the channel name "4tv News" and the country code "IN". Fix is to call the channel "4TV News".

Fixed

File: data/channels.csv [hunk 1]
```diff
@@ -110,119 +110,119 @@ id,name,alt_names,network,owners,country,subdivision,city,broadcast_area,languag
...
+4050Channel.th,4050 Channel,,,,TH,,Bangkok,c/TH,tha,general,FALSE,,,,,https://i.imgur.com/vr5JNVv.png
+40Plus.uk,40+,,,MG Global Entertainment (Europe) Limited,UK,,London,c/UK,eng,xxx,TRUE,2019-04-24,,,https://www.televisionx.com/,https://i.imgur.com/DY9chsN.png
+45TV.hn,45 TV,,,,HN,,La Ceiba,c/HN,spa,news,FALSE,,,,https://45tvhn.com/,https://i.imgur.com/SjUnQlg.png
+48Hours.us,48 Hours,,Pluto TV,,US,,West Hollywood,c/US,eng,series,FALSE,,,,https://pluto.tv/en/live-tv/48-hours,https://i.imgur.com/Q7THHhX.png
+48NewsUK.pk,48 News UK,,,,PK,,London,c/PK,eng,news,FALSE,,,,,https://i.imgur.com/poj9KAI.jpeg
 4BC.au,4BC,,,,AU,,Brisbane,c/AU,eng,news,FALSE,,,,https://www.4bc.com.au/,https://i.imgur.com/VAjzpGw.png
-4DmasNoticiasTV.py,4DmasNoticias TV,4DmásNoticias TV,,,PY,,,c/PY,spa,news,FALSE,,,,,https://i.ibb.co/1fb5BtN/unnamed.png
-4E.gr,4E,,,,GR,,,c/GR,ell,,FALSE,1993-12-16,,,https://tv4e.gr/,https://i.imgur.com/Na0UCV7.png
+4DmasNoticiasTV.py,4DMas Noticias TV Noticias TV,4Dmás Noticias TV,,,PY,,Asuncion,c/PY,spa,news,FALSE,,,,https://4dmasnoticias.com.py/,https://i.ibb.co/1fb5BtN/unnamed.png
```
Comment (BellezaEmporium) [inline data/channels.csv:203]:
```suggestion
4DmasNoticiasTV.py,4DMas Noticias TV,4Dmás Noticias TV,,,PY,,Asuncion,c/PY,spa,news,FALSE,,,,https://4dmasnoticias.com.py/,https://i.ibb.co/1fb5BtN/unnamed.png

```

**Review by BellezaEmporium — DISMISSED** [review]
LGTM, twice.

**Review by AntiPontifex — COMMENTED** [review]
Approve me

**Comment by BellezaEmporium** [comment]
Merge conflict on. Will check when i'm in front of my 'puter

**Comment by BellezaEmporium** [comment]
As of now i'll ask to *refrain* from adding further changes, the time for us to solve the merge conflicts that are in there.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by BellezaEmporium** [comment]
data/channels.csv
 345    "92NewsUK.pk" must be derived from the channel name "92 News UK" and the country code "UK"
 414    "AagTV.pk" must be derived from the channel name "AAG TV" and the country code "PK"
 431    "AathavanTV.in" must be derived from the channel name "Aathavan TV" and the country code "UK"
 433    "AAVISIE.nl" must be derived from the channel name "AAvisie TV" and the country code "NL"
 586    "ABXplore.be" has an invalid languages "fre"
 779    "AflamTV.ma" has an invalid languages "ber"
 229    54UnityTV.th: "owners[0]" with value "D One Solution Co.,Ltd" fails to match the required pattern: /^[^",]+$/
 639    ActionNewsJaxNow.us: "owners[0]" with value "Hoffman Communications, Inc" fails to match the required pattern: /^[^",]+$/

**Comment by AntiPontifex** [comment]
Tried to add fixes; message, Lane is received. Notify me when it is ok to proceed. Thanks.

**Review by BellezaEmporium — APPROVED** [review]
LGTM, and this time, it's all fixed.

**Review by AntiPontifex — COMMENTED** [review]
As you record, LGTM

**Comment by AntiPontifex** [comment]
I seek your approval...

**Comment by AntiPontifex** [comment]
Is it possible to add reviewers, Lane?

**Review by Sphinxroot — APPROVED** [review]

**Review by AntiPontifex — COMMENTED** [review]
Thank you, Sphinxroot :) Hope all is well with you.

**Comment by BellezaEmporium** [comment]
Approval not confirmed sadly. Will need another reviewer on the check.

**Comment by AntiPontifex** [comment]
Copy that

**Review by Alstruit — APPROVED** [review]

**Comment by AntiPontifex** [comment]
Thanks to all involved!

**Pull request merged**